### PR TITLE
Update instructions for RHEL

### LINKED ISSUE #16030: Incorrect package referenced for ASP.NET Core Runtime
In the ASP.NET Core Runtime section the package installed via YUM is the .NET Core Runtime.  The package should be "rh-dotnet30-aspnetcore-runtime-3.0", therefore:

yum install rh-dotnet30-aspnetcore-runtime-3.0 -y

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 666b79de-bce8-4180-3ddf-b1806d3f3605
* Version Independent ID: 5719761e-925b-fd17-9fb1-bafcb03214fe
* Content: [Install .NET Core on Linux RHEL 7 package manager - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-rhel7#feedback)
* Content Source: [docs/core/install/linux-package-manager-rhel7.md](https://github.com/dotnet/docs/blob/master/docs/core/install/linux-package-manager-rhel7.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

Fix package asp.net core RHEL7 and add links to official docs

Fixes #16030
Contributes to #15948

### CODE REVIEW

**Comment by gewarren** [comment]
I think @mairaw is your assigned reviewer.

File: docs/core/install/linux-package-manager-rhel7.md [hunk 1]
```diff
@@ -46,3 +46,7 @@ subscription-manager repos --enable=rhel-7-server-dotnet-rpms
 yum install rh-dotnet30-dotnet-runtime-3.0 -y
 scl enable rh-dotnet30 bash
 ```
+
+## See also
+
+- For more information on installing .NET Core 3.0 with RHEL, see [Using .NET Core 3.0 on Red Hat Enterprise Linux 7](https://access.redhat.com/documentation/en-us/net_core/3.0/html/getting_started_guide/gs_install_dotnet).
```
Comment (mairaw) [inline docs/core/install/linux-package-manager-rhel7.md]: Typically our see alsos are just links

```suggestion
- [Using .NET Core 3.0 on Red Hat Enterprise Linux 7](https://access.redhat.com/documentation/en-us/net_core/3.0/html/getting_started_guide/gs_install_dotnet)
```

File: docs/core/install/linux-package-manager-rhel81.md [hunk 2]
```diff
@@ -43,3 +46,7 @@ After registering with the Subscription Manager, you're ready to install and ena
 sudo dnf install dotnet-runtime-3.0
 scl enable dotnet-runtime-3.0 bash
 ```
+
+## See also
+
+- For more information on installing .NET Core 3.0 with RHEL, see [Using .NET Core 3.0 on Red Hat Enterprise Linux 8](https://access.redhat.com/documentation/en-us/net_core/3.0/html/getting_started_guide_for_rhel_8/gs_install_dotnet).
```
Comment (mairaw) [inline docs/core/install/linux-package-manager-rhel81.md]: ditto

```suggestion
- [Using .NET Core 3.0 on Red Hat Enterprise Linux 8](https://access.redhat.com/documentation/en-us/net_core/3.0/html/getting_started_guide_for_rhel_8/gs_install_dotnet)
```

**Review by mairaw — APPROVED** [review]
LGTM. Left some comments for you to consider. Plus, should you update ms.date?

**Pull request merged**